e2e/deploy-rook.go

```diff
@@ -79,12 +79,16 @@ func deployOperator(c kubernetes.Interface) {
 func deployCluster(c kubernetes.Interface) {
 	opPath := fmt.Sprintf("%s/%s", rookURL, "cluster-test.yaml")
 	framework.RunKubectlOrDie("create", "-f", opPath)
-	err := waitForDaemonSets("rook-ceph-agent", rookNS, c, deployTimeout)
-	Expect(err).Should(BeNil())
+
+	// After rook-ceph v1.1.0, flex driver is disabled. We don't need to wait for rook-ceph-agent.
+	if RookVersion != "master" && RookVersion <= "v1.1.0" {
```

Comment:
sorry I missed one more thing, this will not  hold good for beta version also like `v1.1.0-beta.1` we need to do extract Semantic version and check it

Reply:
hey, I apologize I didn't notice these... I'll extract v1.1.0 part and check it. Thanks again!

Reply:
> sorry I missed one more thing, this will not hold good for beta version also like `v1.1.0-beta.1` we need to do extract Semantic version and check it

Done! but i wonder that 
I wrote version comparing code in util.go, however how do you think about using go-version libary? (https://github.com/hashicorp/go-version) I'm now sure i could import new third-party library. If you doesn't care I will fix up to use that. (in that case need add to /vendor directory?)